src/app/clusters/refrigerator-alarm-server/refrigerator-alarm-server.h

```diff
@@ -31,15 +31,21 @@ class RefrigeratorAlarmServer
 
     EmberAfStatus GetMaskValue(chip::EndpointId endpoint, chip::BitMask<chip::app::Clusters::RefrigeratorAlarm::AlarmMap> * mask);
     EmberAfStatus GetStateValue(chip::EndpointId endpoint, chip::BitMask<chip::app::Clusters::RefrigeratorAlarm::AlarmMap> * state);
+    EmberAfStatus GetSupportedValue(chip::EndpointId endpoint,
+                                    chip::BitMask<chip::app::Clusters::RefrigeratorAlarm::AlarmMap> * suppported);
 
-    // Whenever there is change on Mask we should change State accordingly.
+    // A change in supported value will result in a corresponding change in state.
```

Comment:
This does not seem to belong here. Should this be about setting the mask (as previously)

Reply:
Done